docs/explanation/index.md

```diff
@@ -5,3 +5,5 @@ The explanation provides background knowledge for studying ehrQL.
 * [Using ehrQL in OpenSAFELY projects](using-ehrql-in-opensafely-projects.md)
 * [Running ehrQL](running-ehrql.md)
 * [Using the measures framework](measures.md)
+* [Selecting populations for study](selecting-populations-for-study.md)
+* [The OpenSAFELY VS Code extension](vscode-extension.md)
```

Comment:
I think this was covered in https://github.com/opensafely-core/ehrql/pull/2330, hopefully it will magically disappear on merge